src/test/java/edu/gwu/csci6212/MinimumStepsChessKnightTest.java

```diff
@@ -0,0 +1,56 @@
+package edu.gwu.csci6212;
+
+public class MinimumStepsChessKnightTest {
```

Comment:
As mentioned in WhatsApp, definitely would be nice to see JUnit assertions here.